Makefile

```diff
@@ -70,6 +70,14 @@ CURL ?= curl
 OC ?= oc
 OCM ?= ocm
 
+# Set os specific variables
+ifeq ($(shell uname -s | tr A-Z a-z), darwin)
+    PGHOST:="127.0.0.1"
```

Comment:
nitpick: this line is not aligned with others?

Reply:
fixed this just now. I added a comment about removing this though in this thread: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1602#discussion_r1121711662